osu.Game/Database/DatabaseBackedStore.cs

```diff
@@ -19,6 +22,22 @@ public abstract class DatabaseBackedStore
         private readonly ThreadLocal<OsuDbContext> queryContext;
 
         /// <summary>
+        /// Refresh an instance potentially from a different thread with a local context-tracked instance.
+        /// </summary>
+        /// <param name="obj"></param>
+        /// <param name="lookupSource"></param>

```

Comment:
Would be nice to have some explanation here

Reply:
yep, will populate these.